bootstrap/bootstrap

```diff
@@ -776,9 +776,7 @@ SunOS)
 	need_bsd_install=yes
 	use_bsdinstall=yes
 
-	# Use native nawk and sed if available on newer illumos.
-	/usr/bin/nawk --version 2>/dev/null | grep "awk version " >/dev/null
-	if [ $? -eq 0 ]; then
+	if [ -x "/usr/bin/nawk" ]; then
```

Comment:
I'm not 100% sure this is correct, fuzzy memory says there was a nawk on Solaris 9 that pointed to the XPG4 awk which was unsufficient in some way.  I don't recall all of the details as this was over 20 years ago now, but it's possible this was due to some of the tricker parts of the wrappers which are now somewhat obsolete with cwrappers (though are still used in bootstrap).

I guess it depends on if we want to keep supporting Solaris 9 and older.

Reply:
> I'm not 100% sure this is correct, fuzzy memory says there was a nawk on Solaris 9 that pointed to the XPG4 awk which was unsufficient in some way. I don't recall all of the details as this was over 20 years ago now, but it's possible this was due to some of the tricker parts of the wrappers which are now somewhat obsolete with cwrappers (though are still used in bootstrap).

It seems to point to XPG4 awk on Solaris 10 too.